include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -1704,19 +1720,19 @@ class charge_distribution_surface<Lyt, false> : public Lyt
      * @param history_mode charge_distribution_history::NEGLECT if the information (local electrostatic energy) of the
      * previous charge distribution is used to make the update more efficient, charge_distribution_history::CONSIDER
      * otherwise.
-     * @param engine exhaustive_sidb_simulation_engine::EXGS if `ExGS``should be used,
-     * exhaustive_sidb_simulation_engine::QUICKEXACT for `QuickExact`.
+     * @param engine exact_sidb_simulation_engine::EXGS if `ExGS``should be used,
+     * exact_sidb_simulation_engine::QUICKEXACT for `QuickExact`.
```

Comment:
```suggestion
     * @param `engine exact_sidb_simulation_engine::EXGS` if *ExGS* should be used,
     * `exact_sidb_simulation_engine::QUICKEXACT` for *QuickExact*.
```

Formatting was broken and inconsistent with previous ones. Please double-check throughout the file.